backend/src/app.ts

```diff
@@ -1,20 +1,24 @@
-import { opine, serveStatic } from 'opine/mod.ts';
+import { HTTPOptions, opine, serveStatic } from 'opine/mod.ts';
```

Comment:
```suggestion
import { opine, serveStatic } from 'opine/mod.ts';
```
Looks like an unused import

Reply:
This is not a unused import. We need this in the lines 14 to 17.
```typescript
const options: HTTPOptions = {
  port: config.port,
  hostname: config.hostname,
};
```